src/cmd-buildextend-live

```diff
@@ -341,6 +342,13 @@ def generate_iso():
                 buf = newbuf
                 karg_area_end = re.search(r'(#+) COREOS_KARG_EMBED_AREA\n', buf)
                 if karg_area_end is not None:
+                    if len(default_kargs) == 0:
+                        default_kargs = buf[karg_area_start.start():karg_area_end.end()]
+                        files_with_karg_embed_areas['.cmdline'] = 0
+                        cmdline = os.path.join(tmpisoroot, '.cmdline')
+                        with open(cmdline, 'w') as fh:
+                            fh.write(default_kargs if len(default_kargs) != 0 else kargs)
```

Comment:
I think we can safely assume that `default_kargs` will never have size 0, because we'll at least always need `coreos.liveiso`.

Reply:
done